disttask/framework/storage/task_table.go

```diff
@@ -602,25 +602,24 @@ func (stm *TaskManager) UpdateFailedSchedulerIDs(taskID int64, replaceNodes map[
 	return err
 }
 
-// AddSubTasks add new batch of subtasks.
-func (stm *TaskManager) AddSubTasks(task *proto.Task, subtasks []*proto.Subtask) error {
```

Comment:
Why here use PauseSubtasks replace AddSubTasks?

Reply:
> Why here use PauseSubtasks to replace AddSubTasks?

AddSubtasks is useless, just remove it